### PR TITLE
Add command for flapping the crabarms, modify the crab arm subsystem …

### PR DESCRIPTION
**Pull request by PrathamRawat**
…and modify RobotMap

### CODE REVIEW

**Review by huiminwu — DISMISSED** [review]
edit your .gitignore. many of these files should not be pushed to master. Also, there are merge conflicts in RobotMap.java

File: src/org/usfirst/frc/team694/robot/subsystems/CrabArm.java [hunk 1]
```diff
@@ -12,7 +12,7 @@
 public class CrabArm extends Subsystem {
     private WPI_VictorSPX leftCrabArmMotor;
     private WPI_VictorSPX rightCrabArmMotor;
-    private SpeedControllerGroup crabArmMotors;
+    private static SpeedControllerGroup crabArmMotors;
```
Comment (TacoTechnica) [inline src/org/usfirst/frc/team694/robot/subsystems/CrabArm.java]: Why is this static? Why can't we use Robot.crabArm.flapPush?

**Review by TacoTechnica — DISMISSED** [review]
Why is the speed controller in the crab arm static?

File: src/org/usfirst/frc/team694/robot/subsystems/CrabArm.java [hunk 2]
```diff
@@ -32,7 +32,16 @@ public void acquire() {
     public void flapOut() {
         crabArmMotors.set(-1 * RobotMap.FLAP_OUT_SPEED);
     }
-
+    
+    public static void flapPush() {
```
Comment (huiminwu) [inline src/org/usfirst/frc/team694/robot/subsystems/CrabArm.java:36]: What is the difference between `flapPush()` and `flapCoast()`?
If the differentiation is necessary, please comment to explain.
If not, remove the redundant code.

**Review by huiminwu — APPROVED** [review]

**Pull request merged**